app-text/hoedown/hoedown-3.0.7.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="Standards compliant, fast, secure markdown processing library in C"
+DESCRIPTION="Hoedown is a revived fork of Sundown, the Markdown parser"
+HOMEPAGE="https://github.com/hoedown/hoedown"
+COPYRIGHT="2014 Xaier Mendez, Devin Torres, and Hoedown authors"
+LICENSE="ISC"
+REVISION="1"
+SOURCE_URI="https://github.com/hoedown/hoedown/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="01b6021b1ec329b70687c0d240b12edcaf09c4aa28423ddf344d2bd9056ba920"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	hoedown$secondaryArchSuffix = $portVersion
+	cmd:hoedown$secondaryArchSuffix = $portVersion
+	cmd:smartypants$secondaryArchSuffix = $portVersion
+	lib:libhoedown$secondaryArchSuffix = ${portVersion%%.*}
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	hoedown${secondaryArchSuffix}_devel
+	devel:libhoedown$secondaryArchSuffix = ${portVersion%%.*}
+	"
+REQUIRES_devel="
+	hoedown$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc$secondaryArchSuffix 
+	"
+
+TEST_REQUIRES="
+	cmd:python3
+	cmd:tidy
+"
```

Comment:
Need to check this, but looks fine so far, thanks, only one thing at first sight, add a `tab` before `"` at `line 42`

Reply:
If you push this change things can be merged, check has been fine here (and the TEST) :+1:

Reply:
updated with tab